### PR TITLE
Meet Link Column is added and getEventResource function Created

### PR DESCRIPTION
**Pull request by RamMohanMadhamanchi**
- In one-to-one-scheduler, Meet link column is added
- `getEventResource` is created and used

### CODE REVIEW

File: one-to-one-scheduler/one-to-one-scheduler.gs [hunk 1]
```diff
@@ -368,6 +386,12 @@ function createMeeting(calendar, details) {
     .setGuestsCanSeeGuests(true);
 
     result.eventId = event.getId().replace("@google.com", "");
+    let eventResource = getEventResource(calendar.getId(), result.eventId);
+    if (!eventResource) {
+      result.errorMsg = "Event could not be retrieved - " + calendarId + ", " + meetId;
```
Comment (sailesh-ramayanam) [inline one-to-one-scheduler/one-to-one-scheduler.gs]: Even if `getEventResource` fails, the event has already been created. It is just that we couldn't get its link. So, do not return error. Instead we should have the following.
```js
if (eventResource) {
    result.eventLink = eventResource.hangoutLink;
}
```

**Review by sailesh-ramayanam — CHANGES_REQUESTED** [review]
One minor change

File: one-to-one-scheduler/one-to-one-scheduler.gs (hunk 1 above)
Comment (RamMohanMadhamanchi) [reply one-to-one-scheduler/one-to-one-scheduler.gs]: Updated

**Review by sailesh-ramayanam — APPROVED** [review]

**Comment by RamMohanMadhamanchi** [comment]
[add-on-1-to-1-test-matrix](https://docs.google.com/spreadsheets/d/1yQMdosFKNQ8_p-DD4Krac_9CEpbfPCUUkziR3Oj2eH0/edit?usp=sharing)

**Pull request merged**